src/RpcClient/Utility.cs

```diff
@@ -27,6 +28,21 @@ private static (BigInteger numerator, BigInteger denominator) Fraction(decimal d
             return (numerator, denominator);
         }
 
+        public static UInt160 ToScriptHash(this JObject value)
+        {
+            var addressOrScriptHash = value.AsString();
+
+            foreach (var native in NativeContract.Contracts)
+            {
+                if (addressOrScriptHash.Equals(native.Name, StringComparison.InvariantCultureIgnoreCase) ||
+                    addressOrScriptHash == native.Id.ToString())
```

Comment:
What about non-native contracts?

Reply:
we can't fetch all the smartcontracts

Reply:
Ah, you don't keep mapping of IDs to hashes... OK then. We actually have that inside of neo-go, so I thought it'd be trivial to do the same thing here. But if at least it works for native ones that's good enough for now.